packet.go

```diff
@@ -203,7 +203,7 @@ func (p packetHeader) Version__length() uintptr {
 	return 0
 }
 
-func newPacket(pt packetType, destCid ConnectionId, srcCid ConnectionId, ver VersionNumber, pn uint64, payload []byte) *packet {
+func newPacket(pt packetType, destCid ConnectionId, srcCid ConnectionId, ver VersionNumber, pn uint64, payload []byte, tagLen int) *packet {
```

Comment:
#69 covers this

Reply:
I don't see a relevant change in 69, which doesn't seem to affect packet.go at all